google-sheets/1.0/antora.yml

```diff
@@ -0,0 +1,15 @@
+name: google-sheets-connector
+version: '1.0'
+display_version: 1.0 (Mule 4)
+title: Google Sheets Connector
+nav:
+- modules/ROOT/nav.adoc
+asciidoc:
+  attributes:
+    page-component-desc: xxx
```

Comment:
Needs a one line description. Like:
Synchronizes data and automates business processes between Google Sheets and third-party applications, either on-premises or in the cloud.